### ISSUE TITLE
License for the Assets

### ISSUE THREAD

**Issue by elouin** [solaris-games/solaris#767]
Hello,

I was wondering, what the license for the assets is. Cant seem to find any information on this.

**Comment by mike-eason** [comment]
Hi,

The assets were purchased from here: https://assetstore.unity.com/packages/2d/gui/icons/character-avatar-sci-fi-107693#content

I'll double check if there is a license file and if so add it to the repository.

**Comment by elouin** [comment]
Thanks for the quick answer!
These seem to fall under Unitys asset license. https://unity3d.com/legal/as_terms

It seems like those are only the avatars though, what is with the rest of the assets?

**Comment by mike-eason** [comment]
- The badges fall under a single purchase license.
- The specialist icons fall under a CC BY 3.0 license (basically do whatever you want)
- Everything falls under the same license of this repository, GPL-3.0.

I will add any missing licenses in the next release.

The GPL-3.0 license extends to any use of the code, this basically means that you are free to use and modify the code however anything you do must be open source - anyone can see your modified code and install it for their own purposes.

**Comment by elouin** [comment]
I am just curious what one would have to look out for in a fork/self hosted version of the game(not that i currently plan on doing so).

> I will add any missing licenses in the next release.
Thanks, would be great to know the license for every item in the clients assets folder.

> The GPL-3.0 license extends to any use of the code
It doesn't as far as i know. The case where you run a piece of software on a server(like you would do with Solaris) isn't covered by the GPL(for this case there is the AGPL, see https://www.gnu.org/licenses/why-affero-gpl.en.html). Either way, i was just curious about the licensing on the non code parts of Solaris.

**Comment by mike-eason** [comment]
I'll get that done for you this week.

Understood, my mistake. You're absolutely free to run Solaris on your own server without modifications (as far as I understand the license). 🤝

Just be careful on the UI assets licenses in the client.

**Comment by elouin** [comment]
> I'll get that done for you this week.

Thanks a lot!

> You're absolutely free to run Solaris on your own server without modifications (as far as I understand the license).

I could run a modified version, without releasing the source code as far as I understand the license(I am no lawyer or so).

**Comment by mike-eason** [comment]
I've added the license for the badges, the license for the specialists is already there which I didn't see at first.

I'm not sure what to do for the unity assets other than point you to https://unity3d.com/legal/as_terms as discussed above.

Lastly, the audio assets I couldn't find where I had downloaded them from however I am certain they were free assets. Consider them to be under the repository's license.

Enjoy.
